### DISCUSSION TITLE
[Item] SCI论文阅读模版

### DISCUSSION THREAD

**Discussion by FemiSphere** [windingwind/zotero-better-notes#339] [Note Templates]
###  【模版预览（以一篇nano letter上的文章为例）】：
![image](https://user-images.githubusercontent.com/20103615/216035058-9a547393-8966-4493-ac23-be7c55216c7d.png)
![image](https://user-images.githubusercontent.com/20103615/216035140-08995989-7221-42b7-b2e7-ddb70c68fa71.png)
### 【*模板代码】：
~~~
<h1> <span>&lt;笔记标题&gt;</span></h1>\n <hr/>
</span></h1>\n <h2 style="color: #1B5E20; background-color:#F1F8E9;">💡 Meta Data
</h2> <table>     <tr>         <th style="background-color:#dbeedd;">              <p style="text-align: right">Title  </p>         </th>         <td style="background-color:#dbeedd;">           ${title}         </td>      
  </tr>         <th style="background-color:#f3faf4;">             <p style="text-align: right">Journal  </p>         </th>         <td style="background-color:#f3faf4;">             ${publicationTitle}&nbsp;<em>(${DOI})</em>       
  </td>     </tr>     <tr>         <th style="background-color:#dbeedd;">             <p style="text-align: right">Authors </p>         </th>         <td style="background-color:#dbeedd;">       ${authors}         </td>  
 </tr>         <th style="background-color:#f3faf4;">             <p style="text-align: right">Pub.date </p>         </th>         <td style="background-color:#f3faf4;">             ${date}       </td>       
 </table>
</span></h1>\n <h2 style="color:  #E65100; background-color:  #FFF8E1;">📜 研究背景 & 基础 & 目的</h2> <hr/>
<p></p>
</span></h1>\n <h2 style="color:#2E7D32; background-color:    #F1F8E9;">&#x1F4CA 研究内容</h2> <hr/>
<p></p>
</span></h1>\n <h2 style="color:#4A148C; background-color:    #F5F5F5;">🚩 研究结论</h2> <hr/>
<p></p>

</span></h1>\n <h2 style="color: #006064; background-color:   #E0F7FA;">📌 感想 & 疑问</h2> <hr/>
<p></p>
</span></h1>\n <h2 style="color:#1565C0; background-color:   #E1F5FE;">🔬 研究方法</h2> <hr/>
<h3>  <span><span style="color:  #1565C0">药品：</span></span></h3>
<h3>  <span style="color: #1565C0">表征：</span></h3>
<h3>  <span style="color: #1565C0">其他：</span></h3>
~~~

**Comment by Yang-Yang-Aglaia** [comment]
why don't I use it in Zotero? The problem shows that "Template [Item] Paper阅读笔记 Error: ReferenceError: title is not defined".

**Reply by zhangyin117** [reply]
Have you resolve this problem

**Reply by xysifang** [reply]
The same problem occurs when I use it.“Template [Item] Paper阅读笔记 Error: ReferenceError: title is not defined".

**Reply by FemiSphere** [reply]
印象里我似乎见过这个错误，但是后面我也忘记了自己是怎么解决的了，最近太忙了抱歉没时间复现这个问题，大家可以尝试（1）更新zotero版本or zotero-better-notes版本；（2）稍微更改一下这个代码，具体的操作视频也许可以参考一下这个b站视频：https://www.bilibili.com/video/BV1Fg411H7ZH/?spm_id_from=333.337.search-card.all.click&vd_source=7e2eff77485226543201ad0311af1ee6

**Reply by xysifang** [reply]
回看了视频，还是不知道怎么更改，希望能够指教一二，感谢！

**Reply by Felix-fz** [reply]
可以看下我的解决方案：https://www.bilibili.com/video/BV1H84y1E71B/?vd_source=1f88463fe3c8243881e770e65fcbaa11

**Reply by ZacharyCao** [reply]
不需要修改，需要抓取元数据的模板，命名前要用[Item]，而不能用[Text]，可以看看你的是不是这个问题

**Reply by Na2FeO4** [reply]
把title改成${topItem.getField('title')}或许可以

**Reply by rosie-law** [reply]
> 不需要修改，需要抓取元数据的模板，命名前要用[Item]，而不能用[Text]，可以看看你的是不是这个问题

我通过这个方式可以了！好神奇（小白并不明白原理）感谢~~

**Reply by BrunchTea** [reply]
```html
<h1><span>&lt;笔记标题&gt;</span></h1>\n
<hr/>
</span></h1>\n <h2 style="color: #1B5E20; background-color:#F1F8E9;">💡 Meta Data
</h2>
<table>
    <tr>
        <th style="background-color:#dbeedd;"><p style="text-align: right">Title </p></th>
        <td style="background-color:#dbeedd;"> ${topItem.getField('title')}</td>
    </tr>
    <th style="background-color:#f3faf4;"><p style="text-align: right">Journal </p></th>
    <td style="background-color:#f3faf4;"> ${topItem.getField('publicationTitle')}&nbsp;<em>(${topItem.getField('DOI')})</em>
    </td>
    </tr>
    <tr>
        <th style="background-color:#dbeedd;"><p style="text-align: right">Authors </p></th>
        <td style="background-color:#dbeedd;"> ${topItem.getField('authors')}</td>
    </tr>
    <th style="background-color:#f3faf4;"><p style="text-align: right">Pub.date </p></th>
    <td style="background-color:#f3faf4;"> ${topItem.getField('date')}</td>
</table>
</span></h1>\n <h2 style="color:  #E65100; background-color:  #FFF8E1;">📜 研究背景 & 基础 & 目的</h2>
<hr/>
<p></p>
</span></h1>\n <h2 style="color:#2E7D32; background-color:    #F1F8E9;">&#x1F4CA 研究内容</h2>
<hr/>
<p></p>
</span></h1>\n <h2 style="color:#4A148C; background-color:    #F5F5F5;">🚩 研究结论</h2>
<hr/>
<p></p>

</span></h1>\n <h2 style="color: #006064; background-color:   #E0F7FA;">📌 感想 & 疑问</h2>
<hr/>
<p></p>
</span></h1>\n <h2 style="color:#1565C0; background-color:   #E1F5FE;">🔬 理论推导</h2>
<hr/>
```
即可

**Reply by 18827977838** [reply]
wonderful!!

**Reply by yuanlifang** [reply]
thank you

**Reply by qianxun995** [reply]
thank you

**Reply by wcc910921** [reply]
> > 不需要修改，需要抓取元数据的模板，命名前要用[Item]，而不能用[Text]，可以看看你的是不是这个问题
> 
> 我通过这个方式可以了！好神奇（小白并不明白原理）感谢~~

yes yes it's real!!!!!! thanks for saving my life!

![image](https://github.com/windingwind/zotero-better-notes/assets/43943681/e65cb0df-c428-4973-83e5-aa1454f0c4b5)

**Comment by Kaergo** [comment]
插入模板的时候显示没有定义title是为什么？

**Reply by FemiSphere** [reply]
印象里我似乎见过这个错误，但是后面我也忘记了自己是怎么解决的了，最近太忙了抱歉没时间复现这个问题，大家可以尝试（1）更新zotero版本or zotero-better-notes版本；（2）稍微更改一下这个代码，具体的操作视频也许可以参考一下这个b站视频：https://www.bilibili.com/video/BV1Fg411H7ZH/?spm_id_from=333.337.search-card.all.click&vd_source=7e2eff77485226543201ad0311af1ee6

**Reply by Felix-fz** [reply]
可以看下我的解决方案：https://www.bilibili.com/video/BV1H84y1E71B/?vd_source=1f88463fe3c8243881e770e65fcbaa11

**Reply by ZacharyCao** [reply]
不需要修改，需要抓取元数据的模板，命名前要用[Item]，而不能用[Text]，可以看看你的是不是这个问题

**Reply by keji1115** [reply]
请问这个问题解决了吗？能不能告知一下，万分感谢！

**Comment by Yang-Yang-Aglaia** [comment]
我不知道你是因为啥，但我自己当时是因为把“必需字节”漏了。Sent from my iPhoneOn Feb 27, 2023, at 00:31, xysifang ***@***.***> wrote:﻿
回看了视频，还是不知道怎么更改，希望能够指教一二，感谢！

**Reply by Felix-fz** [reply]
可以看下我的解决方案：https://www.bilibili.com/video/BV1H84y1E71B/?vd_source=1f88463fe3c8243881e770e65fcbaa11

**Reply by xysifang** [reply]
> 可以看下我的解决方案：https://www.bilibili.com/video/BV1H84y1E71B/?vd_source=1f88463fe3c8243881e770e65fcbaa11

可不可以分享下，看不清楚代码信息，小白不知缺少了哪里，蟹蟹呀

**Reply by Felix-fz** [reply]
你好，我在视频简介中更新了开源地址，您可以参考

**Reply by ZacharyCao** [reply]
不需要修改，需要抓取元数据的模板，命名前要用[Item]，而不能用[Text]，可以看看你的是不是这个问题

**Comment by FeijvZhang** [comment]
如果“Error: ReferenceError: title is not defined“，可将代码中 从 `<table>` 到`</table>` 部分改为以下。仅供参考。
```
<table>
    <tr>
        <th style="background-color:#dbeedd;">
            <p style="text-align: left">Title </p>
        </th>
        <td style="background-color:#dbeedd;">${topItem.getField('title')}
        </td>
    </tr>
    <tr>
        <th style="background-color:#f3faf4;">
            <p style="text-align:  left">Journal </p>
        </th>
        <td style="background-color:#f3faf4;">${topItem.getField('publicationTitle')}
        </td>
    </tr>
    <tr>
        <th style="background-color:#dbeedd;">
            <p style="text-align:  left">Authors </p>
        </th>
        <td style="background-color:#dbeedd;">${topItem.getCreators().map((v)=>v.firstName+" "+v.lastName).join("; ")}
        </td>
    </tr>
    <tr>
        <th style="background-color:#f3faf4;">
            <p style="text-align: left">Pub. date </p>
        </th>
        <td style="background-color:#f3faf4;">${topItem.getField('date')}
        </td>
    </tr>
    <tr>
        <th style="background-color:#dbeedd;">
            <p style="text-align: left">DOI </p>
        </th>
        <td style="background-color:#dbeedd;">
        <a href="https://doi.org/${topItem.getField('DOI')}">${topItem.getField('DOI')}</a>     
        </td>
    </tr>
</table>
<p></p>
```

**Reply by xysifang** [reply]
> 如果“Error: ReferenceError: title is not defined“，可将代码中 从 `<table>` 到`</table>` 部分改为以下。仅供参考。
> 
> ```
> <table>
>     <tr>
>         <th style="background-color:#dbeedd;">
>             <p style="text-align: left">Title </p>
>         </th>
>         <td style="background-color:#dbeedd;">${topItem.getField('title')}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#f3faf4;">
>             <p style="text-align:  left">Journal </p>
>         </th>
>         <td style="background-color:#f3faf4;">${topItem.getField('publicationTitle')}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#dbeedd;">
>             <p style="text-align:  left">Authors </p>
>         </th>
>         <td style="background-color:#dbeedd;">${topItem.getCreators().map((v)=>v.firstName+" "+v.lastName).join("; ")}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#f3faf4;">
>             <p style="text-align: left">Pub. date </p>
>         </th>
>         <td style="background-color:#f3faf4;">${topItem.getField('date')}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#dbeedd;">
>             <p style="text-align: left">DOI </p>
>         </th>
>         <td style="background-color:#dbeedd;">
>         <a href="https://doi.org/${topItem.getField('DOI')}">${topItem.getField('DOI')}</a>     
>         </td>
>     </tr>
> </table>
> <p></p>
> ```

谢谢你

**Reply by LiSonglin-xidian** [reply]
可以使用

**Reply by 1057337811** [reply]
“从 到 部分改到以下” 是什么意思求！！！ 可以告诉我这些table 插到哪里吗?

**Reply by FeijvZhang** [reply]
> “从 到 部分改到以下” 是什么意思求！！！ 可以告诉我这些table 插到哪里吗?

在模板代码中找到  table  的位置和 斜杠table 的位置，把这两个位置之间原本的代码（包含两个table）删除，在相同位置换成这个回复中的代码，模板就能正常使用。希望能帮到你。

**Reply by ThreeStones1029** [reply]
> 如果“Error: ReferenceError: title is not defined“，可将代码中 从 `<table>` 到`</table>` 部分改为以下。仅供参考。
> 
> ```
> <table>
>     <tr>
>         <th style="background-color:#dbeedd;">
>             <p style="text-align: left">Title </p>
>         </th>
>         <td style="background-color:#dbeedd;">${topItem.getField('title')}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#f3faf4;">
>             <p style="text-align:  left">Journal </p>
>         </th>
>         <td style="background-color:#f3faf4;">${topItem.getField('publicationTitle')}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#dbeedd;">
>             <p style="text-align:  left">Authors </p>
>         </th>
>         <td style="background-color:#dbeedd;">${topItem.getCreators().map((v)=>v.firstName+" "+v.lastName).join("; ")}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#f3faf4;">
>             <p style="text-align: left">Pub. date </p>
>         </th>
>         <td style="background-color:#f3faf4;">${topItem.getField('date')}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#dbeedd;">
>             <p style="text-align: left">DOI </p>
>         </th>
>         <td style="background-color:#dbeedd;">
>         <a href="https://doi.org/${topItem.getField('DOI')}">${topItem.getField('DOI')}</a>     
>         </td>
>     </tr>
> </table>
> <p></p>
> ```

可以，感谢大佬

**Reply by GeMSeven** [reply]
感谢，问题解决了

**Reply by Serendipity-LJ** [reply]
请问开头meta data表格里的第二行journal和最后一行doi没有内容显示 怎么设置呀？

**Comment by syy-hei** [comment]
![image](https://user-images.githubusercontent.com/78795085/222453751-b7f674a3-61d3-4840-81f4-4b538f67375a.png)
为啥我用上面的代码编写到模板中，获取到的标签下面这样的呢？
![image](https://user-images.githubusercontent.com/78795085/222454164-08414895-1f52-4f28-ab93-d23aff3f3389.png)
返回了一大串 object，下面是我写在模板中的代码，有大佬能够帮我看看是什么原因吗，
![image](https://user-images.githubusercontent.com/78795085/222454480-e5689889-3dd7-4160-841c-d4ebf841413e.png)
我使用这个代码的目的就是想，可以在笔记中添加项目的标签
![image](https://user-images.githubusercontent.com/78795085/222454909-0b1efd05-04bc-4489-a418-baa90ed80cd3.png)

**Comment by syy-hei** [comment]
image
为啥我用上面的代码编写到模板中，获取到的标签下面这样的呢？
image
返回了一大串 object，下面是我写在模板中的代码，有大佬能够帮我看看是什么原因吗，
image
我使用这个代码的目的就是想，可以在笔记中添加项目的标签

**Comment by FemiSphere** [comment]
【您好，马心远已经收到信息，感谢您百忙之中垂询】Hello,mxy have received information。Thank you for your busy schedule Inquiries

**Comment by lyk7539511** [comment]
可否禁止MetaData表格第一列的自动换行，我尝试使用了多种方法包括nobr、inline等，都无法在zotero中生效

**Comment by FemiSphere** [comment]
【您好，马心远已经收到信息，感谢您百忙之中垂询】Hello,mxy have received information。Thank you for your busy schedule Inquiries

**Comment by FantasticRhythm** [comment]
我的问题是Template CS Error: SyntaxError: missing formal parameter

**Reply by max-seu** [reply]
请问解决了吗

**Reply by max-seu** [reply]
在创建模板时，模板名称需要设置对，前面要加[Item]

**Reply by zhit123** [reply]
> 在创建模板时，模板名称需要设置对，前面要加[Item]
谢谢你

**Comment by thisAnyaLee** [comment]
简单修改下:
```
<h1>${topItem.getField("title")}</h1>
<hr>
<h2 style="color: #1B5E20; background-color:#F1F8E9;">💡 Meta Data</h2>
<table>
    <tr>
        <th style="background-color:#dbeedd;">
            <p style="text-align: left">Title </p>
        </th>
        <td style="background-color:#dbeedd;">${topItem.getField('title')}
        </td>
    </tr>
    <tr>
        <th style="background-color:#f3faf4;">
            <p style="text-align:  left">Journal </p>
        </th>
        <td style="background-color:#f3faf4;">${topItem.getField('publicationTitle')}
        </td>
    </tr>
    <tr>
        <th style="background-color:#dbeedd;">
            <p style="text-align:  left">Authors </p>
        </th>
        <td style="background-color:#dbeedd;">${topItem.getCreators().map((v) => v.firstName + " " + v.lastName).join(";")}
        </td>
    </tr>
    <tr>
        <th style="background-color:#f3faf4;">
            <p style="text-align: left">Pub. date </p>
        </th>
        <td style="background-color:#f3faf4;">${topItem.getField('date')}
        </td>
    </tr>
    <tr>
        <th style="background-color:#dbeedd;">
            <p style="text-align: left">DOI </p>
        </th>
        <td style="background-color:#dbeedd;">
            <a href="https://doi.org/${topItem.getField('DOI')}">${topItem.getField('DOI')}</a>
        </td>
    </tr>
</table>

<h2 style="color:  #E65100; background-color:  #FFF8E1;">📜 研究背景 & 启发 & 目的</h2>
<hr>
<h3 style="color:  #E65100;">Background</h3>
<p></p>
<h3 style="color:  #E65100;">Inspiration</h3>
<p></p>
<h3 style="color:  #E65100;">Proposal</h3>
<p></p>

<h2 style="color:#1565C0; background-color:#E1F5FE;">&#x1F4CA 研究方法</h2>
<hr>
<h3 style="color:  #1565C0">XX</h3>
<p></p>
<h3 style="color: #1565C0">XX</h3>
<p></p>
<h3 style="color: #1565C0">XX</h3>
<p></p>

<h2 style="color: #2E7D32; background-color:#F1F8E9;">🔬 实验结果</h2>
<hr>
<p></p>

<h2 style="color: #4A148C; background-color:#F5F5F5;">🚩 研究结论</h2>
<hr>
<h3 style="color:  #4A148C;">Contribution</h3>
<p></p>
<h3 style="color:  #4A148C;">Future work</h3>
<p></p>

<h2 style="color: #006064; background-color:#E0F7FA;">📌 感想 & 疑问</h2>
<hr>
<p></p>
```

**Reply by Humble-LiuAo** [reply]
thanks bro!

**Reply by zhit123** [reply]
> 简单修改下:
> 
> ```
> <h1>${topItem.getField("title")}</h1>
> <hr>
> <h2 style="color: #1B5E20; background-color:#F1F8E9;">💡 Meta Data</h2>
> <table>
>     <tr>
>         <th style="background-color:#dbeedd;">
>             <p style="text-align: left">Title </p>
>         </th>
>         <td style="background-color:#dbeedd;">${topItem.getField('title')}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#f3faf4;">
>             <p style="text-align:  left">Journal </p>
>         </th>
>         <td style="background-color:#f3faf4;">${topItem.getField('publicationTitle')}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#dbeedd;">
>             <p style="text-align:  left">Authors </p>
>         </th>
>         <td style="background-color:#dbeedd;">${topItem.getCreators().map((v) => v.firstName + " " + v.lastName).join(";")}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#f3faf4;">
>             <p style="text-align: left">Pub. date </p>
>         </th>
>         <td style="background-color:#f3faf4;">${topItem.getField('date')}
>         </td>
>     </tr>
>     <tr>
>         <th style="background-color:#dbeedd;">
>             <p style="text-align: left">DOI </p>
>         </th>
>         <td style="background-color:#dbeedd;">
>             <a href="https://doi.org/${topItem.getField('DOI')}">${topItem.getField('DOI')}</a>
>         </td>
>     </tr>
> </table>
> 
> <h2 style="color:  #E65100; background-color:  #FFF8E1;">📜 研究背景 & 启发 & 目的</h2>
> <hr>
> <h3 style="color:  #E65100;">Background</h3>
> <p></p>
> <h3 style="color:  #E65100;">Inspiration</h3>
> <p></p>
> <h3 style="color:  #E65100;">Proposal</h3>
> <p></p>
> 
> <h2 style="color:#1565C0; background-color:#E1F5FE;">&#x1F4CA 研究方法</h2>
> <hr>
> <h3 style="color:  #1565C0">XX</h3>
> <p></p>
> <h3 style="color: #1565C0">XX</h3>
> <p></p>
> <h3 style="color: #1565C0">XX</h3>
> <p></p>
> 
> <h2 style="color: #2E7D32; background-color:#F1F8E9;">🔬 实验结果</h2>
> <hr>
> <p></p>
> 
> <h2 style="color: #4A148C; background-color:#F5F5F5;">🚩 研究结论</h2>
> <hr>
> <h3 style="color:  #4A148C;">Contribution</h3>
> <p></p>
> <h3 style="color:  #4A148C;">Future work</h3>
> <p></p>
> 
> <h2 style="color: #006064; background-color:#E0F7FA;">📌 感想 & 疑问</h2>
> <hr>
> <p></p>
> ```

"Error: ReferenceError: topItem is not defined" why?

**Comment by Serendipity-LJ** [comment]
请问开头meta data表格里的第二行journal和最后一行doi没有内容显示 怎么设置呀？

**Comment by baseoline** [comment]
一直显示topitem is not defied怎么办

**Reply by zen-tang** [reply]
模板名的前的[Item]首字母一定要大写！！！！

**Reply by muyuan-ke** [reply]
> 模板名的前的[Item]首字母一定要大写！！！！

请问这个[Item]是在哪里呀，小白找没找到

**Reply by keji1115** [reply]
大写了也不能用，您现在用的是哪个版本的呀？

**Reply by zen-tang** [reply]
![image](https://github.com/windingwind/zotero-better-notes/assets/64677034/07323d24-7bd5-4049-8f0d-ec62e4e7af1d)
版本 6.0.26 已经能成功用上模板了

**Reply by muyuan-ke** [reply]
感谢，可以了

**Reply by studybutcher** [reply]
> 模板名的前的[Item]首字母一定要大写！！！！

！！！！！！！！thanks，my bro!!

**Comment by muyuan-ke** [comment]
打扰了，有个笨办法就是把topItem.getfield删掉，参照讨论区sci模板改成title，我试了这样是可以预览的发自我的荣耀手机-------- 原始邮件 --------发件人： keji1115 ***@***.***>日期： 2023年6月4日周日 10:26收件人： windingwind/zotero-better-notes ***@***.***>抄送： muyuan-ke ***@***.***>, Comment ***@***.***>主    题： Re: [windingwind/zotero-better-notes] [Item] SCI论文阅读模版 (Discussion #339)
请问这个问题解决了吗？能不能告知一下，万分感谢！

**Comment by sprouting-pixel** [comment]
![image](https://github.com/windingwind/zotero-better-notes/assets/79639932/f1950c2c-715a-4b8f-ba19-9357da792037)
ni你好这种情况怎么解决呀

**Reply by selfpowerful** [reply]
If zotero is version 6.*, better-notes should be version 0.8.9 and below.

**Comment by swenqing** [comment]
有没有人直接给一下可以的源码啊，我zotero6.026,版本，zotero better notes是1.0.4版本的，试了上面所有的代码都不行啊

**Reply by selfpowerful** [reply]
看我上一层的回答

**Comment by Olivia-Ivy** [comment]
![image](https://github.com/windingwind/zotero-better-notes/assets/141628492/c529d151-0dad-4274-ba41-bad109a93399)
为什么我前面表格里的数据抓取不出来？求大佬讲解

**Reply by selfpowerful** [reply]
你这个代码运行成功了啊，尝试以下的方法
1 检查zotero和better notes的版本是否对应，0.8.9以上的BN对应zotero7
2 条目信息是否完整，包含对应的期刊作者等

**Reply by shopping1116** [reply]
<h1><strong>🍓 精读笔记 📍</strong>${title}</h1> 
<p><i><em>Author: ${authors}</em></i></p>
<p style="background-color: #efe3da;">🔴Journal: ${publicationTitle}</p>
<p style="background-color: #efe3da;">🔴JCR分区: ${archive}   🔴IF: ${callNumber}</p> 
<p style="background-color: #efe3da;">🔴中科院分区: ${archiveLocation} </p>
<hr class="hr1"/>
<h2 style="color:red; background-color: #efe3da;"><strong>🔎 研究背景</strong></h2>
<h3  style="background-color: #efe3da;">1. 为什么要研究这个问题？</h3>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
 <h3  style="background-color: #efe3da;"> 2. 所研究的问题目前存在什么样的困难？</h3>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
 <h3  style="background-color: #efe3da;">3. 学者们都做了哪些研究来解决此问题？</h3>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
 <h3  style="background-color: #efe3da;">4. 这些解决方法还有什么不足?</h3>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
<hr class="hr1"/>
<h2 style="color:red; background-color: #efe3da;"><strong>❓ 研究目的</strong></h2>
<pre><code>着重看文章的研究目的是为了要解决什么问题？比如为了进行优化？为了更加贴近现实的加载情况？提高优化的结果？等等</code>  </pre>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
<hr class="hr1"/>
<h2 style="color:red; background-color: #efe3da;"><strong>📝 研究方法</strong></h2>
<pre><code>论文具体采用了什么方法来达到自己的研究目的</code>  </pre>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
<hr class="hr1"/>
<h2 style="color:red; background-color: #efe3da;"><strong>💎 创新点</strong></h2>
<pre><code>论文是提出了新的理论还在原有的理论上，改进了研究方法等</code>  </pre>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
<hr class="hr1"/>
<h2 style="color:red; background-color: #efe3da;"><strong>📝 研究内容</strong></h2>
<pre><code>主要研究了哪几部分内容，算例分析中分析了算了哪些算例？进行了哪些方面分析？</code>  </pre>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
<hr class="hr1"/>
<h2 style="color:red; background-color: #efe3da;"><strong>📑 结论</strong></h2>
<pre><code>根据一些算例分析，具体的到了哪些结论，这些结论又是如何达到了自己的研究目的</code>  </pre>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
<hr class="hr1"/>
<h2 style="color:red; background-color: #efe3da;"><strong>💹 重要的图表</strong></h2>
<pre><code>摘录一些重要得到图表，用于借鉴图表的表达方式，用于自己绘图时借鉴；</br>以及一些与自己的研究有关的图表,写出文章中没有分析出来的结果</br></code>  </pre>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
<hr class="hr1"/>
<h2 style="color:red; background-color: #efe3da;"><strong>💡 启发</strong></h2>
<pre><code>这篇论文与自己的研究内容有哪些相似之处，那些的地方自己可以借鉴；或者有什么可以继续研究的idea</br></code>  </pre>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
<hr class="hr1"/>
<h2 style="color:red; background-color: #efe3da;"><strong>✔️ 摘录内容</strong></h2>
<pre><code>摘录一些自己论文可以会用到的东西，综述部分，结论部分，或者分析部分</br></code>  </pre>
<blockquote>
    <p></p>
    <p></p>
</blockquote>
<hr class="hr1"/>
能否帮忙看看，不能引用模板

**Comment by baseoline** [comment]
模板名称设计为[Item]

 

<h1> <span>&lt;笔记标题&gt;</span></h1>\n <hr/>

</span></h1>\n <h2 style="color: #1B5E20; background-color:#F1F8E9;">💡 Meta Data

</h2> <table>

    <tr>

        <th style="background-color:#dbeedd;">

            <p style="text-align: left">Title </p>

        </th>

        <td style="background-color:#dbeedd;">${topItem.getField('title')}

        </td>

    </tr>

    <tr>

        <th style="background-color:#f3faf4;">

            <p style="text-align:  left">Journal </p>

        </th>

        <td style="background-color:#f3faf4;">${topItem.getField('publicationTitle')}

        </td>

    </tr>

    <tr>

        <th style="background-color:#dbeedd;">

            <p style="text-align:  left">Authors </p>

        </th>

        <td style="background-color:#dbeedd;">${topItem.getCreators().map((v)=>v.firstName+" "+v.lastName).join("; ")}

        </td>

    </tr>

    <tr>

        <th style="background-color:#f3faf4;">

            <p style="text-align: left">Pub. date </p>

        </th>

        <td style="background-color:#f3faf4;">${topItem.getField('date')}

        </td>

    </tr>

    <tr>

        <th style="background-color:#dbeedd;">

            <p style="text-align: left">DOI </p>

        </th>

        <td style="background-color:#dbeedd;">

        <a href=https://doi.org/${topItem.getField('DOI')} <https://doi.org/$%7btopItem.getField('DOI')%7d> >${topItem.getField('DOI')}</a>     

        </td>

    </tr>

</table>

<p></p>

</span></h1>\n <h2 style="color:  #E65100; background-color:  #FFF8E1;">📜 研究背景 & 基础 & 目的</h2> <hr/>

<p></p>

</span></h1>\n <h2 style="color:#2E7D32; background-color:    #F1F8E9;">&#x1F4CA 研究内容</h2> <hr/>

<p></p>

</span></h1>\n <h2 style="color:#4A148C; background-color:    #F5F5F5;">🚩 研究结论</h2> <hr/>

<p></p>

 

</span></h1>\n <h2 style="color: #006064; background-color:   #E0F7FA;">📌 感想 & 疑问</h2> <hr/>

<p></p>

</span></h1>\n <h2 style="color:#1565C0; background-color:   #E1F5FE;">🔬 研究方法</h2> <hr/>

<h3>  <span><span style="color:  #1565C0">药品：</span></span></h3>

<h3>  <span style="color: #1565C0">表征：</span></h3>

<h3>  <span style="color: #1565C0">其他：</span></h3>

 

 

 

 

 

 

发件人: ***@***.*** ***@***.***> 代表 swenqing
发送时间: 2023年7月27日 16:25
收件人: windingwind/zotero-better-notes ***@***.***>
抄送: baseoline ***@***.***>; Comment ***@***.***>
主题: Re: [windingwind/zotero-better-notes] [Item] SCI论文阅读模版 (Discussion #339)

 

有没有人直接给一下可以的源码啊，我zotero6.026,版本，zotero better notes是1.0.4版本的，试了上面所有的代码都不行啊

**Reply by selfpowerful** [reply]
你这个betternotes的版本是对应zotero7的，换成0.8.9或以前的

**Comment by Avocadoisgood** [comment]
为什么我没有办法显示作者呢？
<img width="352" alt="屏幕截图 2024-01-12 092654" src="https://github.com/windingwind/zotero-better-notes/assets/156305531/2804c889-3bfe-4418-b91e-e95b399881bd">

**Comment by fredericky123** [comment]
模板里第一列的单词宽度能否设置为一行显示完呢 即怎么设置宽度
